src/main/java/blackjack/model/card/Pattern.java

```diff
@@ -0,0 +1,9 @@
+package blackjack.model.card;
+
+public enum Pattern {
+
+    SPADE,
+    HEART,
+    DIAMOND,
+    CLOVER;
```

Comment:
```suggestion
    CLOVER,
    ;
```
다음 enum의 추가를 위해서 이렇게 표현하기도 합니다 :)

Reply:
처음 알게 된 표현 방식이네요! 코드에 남겨두기 위해 우선 Pattern enum에만 적용해두었습니다. 👍